src/init.py

```diff
@@ -30,7 +30,6 @@
 import util.write_config as WriteConfig
 import util.read_config as ReadConfig
 
-from PySide2.QtWidgets import QApplication
 
 
 class MyApplication(QtWidgets.QMainWindow):
```

Comment:
won't this give an import error?

Reply:
I don't think so (application already runs without error) because we are already importing whole QtWidgets in a previous line

Reply:
Which line? I do not see it. 🏳️

Reply:
This line. https://github.com/LordAmit/Brightness/blob/9ab45ba18ba6359b90c7d9009a9a76bf41bebce4/src/init.py#L22

Reply:
Damnit github.. I thought it was line 30 was the first line. 🤦‍♂